webapi/src/main/twirl/queries/sparql/v1/addValueVersion.scala.txt

```diff
@@ -200,6 +200,7 @@ DELETE {
             ?newValue <@assertion._1> <@assertion._2> .
         }
 
+        @* TODO: in case of a file value, order is not given *@
```

Comment:
No, the thing we dealt with yesterday was the property's `guiOrder`, not `valueHasOrder`